### PR TITLE
Add the 'large' width option to Paragraphs

### PR DESCRIPTION
**Pull request by codeswan**

### CODE REVIEW

File: src/js/components/Paragraph.js [hunk 1]
```diff
@@ -31,5 +32,6 @@ Paragraph.propTypes = {
   align: PropTypes.oneOf(['start', 'center', 'end']),
   id: PropTypes.string,
   margin: PropTypes.oneOf(['none', 'small', 'medium', 'large']),
-  size: PropTypes.oneOf(['small', 'medium', 'large'])
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
+  width: PropTypes.oneOf(['small', 'medium', 'large'])
```
Comment (ericsoderberghp) [inline src/js/components/Paragraph.js]: Would it work to key the width and max-width off of the 'size' property value? I'm thinking that the max readable text width is relative to the font size, so the larger the font size, the larger the width and max-width.

Comment (codeswan) [inline src/js/components/Paragraph.js]: @ericsoderberghp - I don't think that would be ideal.  I have some cases where it would be preferable to be able to have different text-sized paragraphs be of the same width, and automatically basing paragraph width off of text-size could mess up existing styling for anyone who's already using Grommet, since it wouldn't be backwards-compatible.

Comment (ericsoderberghp) [inline src/js/components/Paragraph.js]: I'm thinking then that we should align widths for other things, like Image, Video, and Quote, to allow pages that combine these elements to achieve a more consistent layout. One option would be to put the width constraint on Box and use full for all of the others, to fill the available width. That would work well for including other things too, like Chart, and Distribution.

Comment (codeswan) [inline src/js/components/Paragraph.js]: @ericsoderberghp I've submitted a separate PR for adding sizing to Box.  I believe this should work as desired, so please review it when you get a chance.  Thanks!

File: src/scss/grommet-core/_objects.paragraph.scss [hunk 2]
```diff
@@ -28,4 +28,9 @@
       font-weight: 600;
     }
   }
+
+  &.paragraph--width-large {
+    width: $inuit-base-spacing-unit * 30;
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.paragraph.scss]: Assuming we will convert to the standard sizing variables when the Box sizing pull request comes in.
